docs/website/docs/guides/ml-frameworks/pytorch.md

```diff
@@ -322,6 +322,132 @@ their values independently at runtime.
         self.value = new_value
     ```
 
+#### :octicons-code-16: Separating parameters during compilation
```

Comment:
nit: could use `octicons-file-symlink-file-16` to match the icon used for https://iree.dev/guides/parameters/ in the nav: https://github.com/iree-org/iree/blob/c86b621665ad8b898c0d0e987a45267f78eb9c3f/docs/website/docs/guides/parameters.md?plain=1#L2

Consistency and repetition across pages can make concepts easier to learn.